src/components/Expenses.js

```diff
@@ -0,0 +1,60 @@
+import ExpenseDate from "./ExpenseDate";
+import ExpenseItem from "./ExpenseItem";
```

Comment:
Do not put all code into this component. Move the expenses back to app.js and pass change expenses to items